impeller/display_list/dl_dispatcher.cc

```diff
@@ -1030,7 +1030,8 @@ void DlDispatcherBase::drawDisplayList(
     save_paint.color = Color(0, 0, 0, opacity);
     GetCanvas().SaveLayer(
         save_paint, skia_conversions::ToRect(display_list->bounds()), nullptr,
-        ContentBoundsPromise::kContainsContents, display_list->total_depth());
+        ContentBoundsPromise::kContainsContents, display_list->total_depth(),
+        display_list->can_apply_group_opacity());
```

Comment:
@flar  Mystery solved as to why this didn't seem to work, this was using the `false` default value.

Reply:
Ah, the flag wasn't being passed along?

Reply:
only in the regular saveLayer case, which is why it sometimes worked, but not from this saveLayer in drawDL


my bad!